src/core/js/router.js

```diff
@@ -101,6 +102,10 @@ define(function(require) {
             }
         },
 
+        navigateToHomeRoute: function() {
+            this.navigate('#', {trigger: true});
```

Comment:
Can we have if (Adapt.router.get('_canNavigate')) {  in there as well?

We've an extension called 'promptOnPageIncomplete'. This'd be quite usefull.